tests/lax_test.py

```diff
@@ -183,6 +183,28 @@ def op_record(op, nargs, dtypes, rng_factory, tol=None):
     op_record("lt", 2, default_dtypes, jtu.rand_small),
 ]
 
+ReducerOpRecord = collections.namedtuple(
+  "ReducerOpRecord", ["op", "reference_op", "init_val", "dtypes", "primitive"]
+)
+
+LAX_REDUCE_OPS = [
+  ReducerOpRecord(lax.add, np.add, 0, default_dtypes, lax.reduce_sum_p),
+  ReducerOpRecord(lax.mul, np.multiply, 1, default_dtypes, lax.reduce_prod_p),
+  ReducerOpRecord(lax.max, np.maximum, 0, float_dtypes + complex_dtypes + int_dtypes, lax.reduce_p),
+  ReducerOpRecord(lax.max, np.maximum, 0, uint_dtypes + bool_dtypes, lax.reduce_max_p),
+  ReducerOpRecord(lax.max, np.maximum, -np.inf, float_dtypes, lax.reduce_max_p),
+  ReducerOpRecord(lax.max, np.maximum, dtypes.iinfo(np.int32).min, [np.int32], lax.reduce_max_p),
+  ReducerOpRecord(lax.max, np.maximum, dtypes.iinfo(np.int64).min, [np.int64], lax.reduce_max_p),
+  ReducerOpRecord(lax.min, np.minimum, np.inf, float_dtypes, lax.reduce_min_p),
+  ReducerOpRecord(lax.min, np.minimum, dtypes.iinfo(np.int32).max, [np.int32], lax.reduce_min_p),
+  ReducerOpRecord(lax.min, np.minimum, dtypes.iinfo(np.int64).max, [np.int64], lax.reduce_min_p),
+  ReducerOpRecord(lax.min, np.minimum, dtypes.iinfo(np.uint32).max, [np.uint32], lax.reduce_min_p),
+  ReducerOpRecord(lax.min, np.minimum, dtypes.iinfo(np.uint64).max, [np.uint64], lax.reduce_min_p),
+  ReducerOpRecord(lax.bitwise_and, np.bitwise_and, dtypes.iinfo(np.uint32).max, [np.uint32], lax.reduce_and_p),
```

Comment:
Could we use `-1` and `uint_dtypes` here? (since `uint(-1)` returns the maximum value)?